src/main/kotlin/com/jetbrains/snakecharm/inspections/SmkMultilineFunctionCallInspection.kt

```diff
@@ -0,0 +1,98 @@
+import com.jetbrains.python.PyTokenTypes
+import com.jetbrains.python.psi.PyCallExpression
+import com.jetbrains.snakecharm.SnakemakeBundle
+import com.jetbrains.snakecharm.lang.psi.SmkRuleOrCheckpointArgsSection
+
+const val TAB = "    "
+
+class SmkMultilineFunctionCallInspection : SnakemakeInspection() {
+    override fun buildVisitor(
+        holder: ProblemsHolder,
+        isOnTheFly: Boolean,
+        session: LocalInspectionToolSession
+    ) = object : SnakemakeInspectionVisitor(holder, session) {
+        override fun visitSmkRuleOrCheckpointArgsSection(st: SmkRuleOrCheckpointArgsSection) {
+            val args = st.argumentList ?: return
+
+            if (st.multilineSectionDefinition()) {
+                return
+            }
+
+            val invalidWhitespaces = mutableListOf<PsiElement>()
+            args.arguments.forEach { psi ->
+                if (psi is PyCallExpression) {
+                    collectNewlinesInMultilineCall(psi, invalidWhitespaces)
+                }
+            }
+
+            invalidWhitespaces.forEach {
+                registerProblem(
+                    it,
+                    SnakemakeBundle.message("INSP.NAME.multiline.func.call"),
+                    ShiftToNextLine(st, invalidWhitespaces)
+                )
+            }
+        }
+    }
+
+    /**
+     * Checks whether there are any whitespaces in [expression] argument list,
+     * if so, checks if it contains new line character,
+     * if so, adds whitespace nodes to [incorrectElements]
+     */
+    private fun collectNewlinesInMultilineCall(
+        expression: PyCallExpression,
+        incorrectElements: MutableList<PsiElement>
+    ) {
+        var element = expression.argumentList?.firstChild
+        while (element != null) {
+            if (element.elementType == TokenType.WHITE_SPACE && element.text.contains("\n")) {
+                incorrectElements.add(element)
+                break
+            }
+            element = element.nextSibling
+        }
+    }
+
+    private class ShiftToNextLine(expr: PsiElement, val incorrectElements: MutableList<PsiElement>) :
+        LocalQuickFixOnPsiElement(expr) {
+
+        override fun getFamilyName() = SnakemakeBundle.message("INSP.NAME.multiline.func.call.fix")
+
+        override fun getText() = familyName
+
+        override fun invoke(project: Project, file: PsiFile, startElement: PsiElement, endElement: PsiElement) {
+            val doc = PsiDocumentManager.getInstance(project).getDocument(file)
+            if (startElement.prevSibling !is PsiWhiteSpace || doc == null) {
+                return
+            }
+            val argumentList = (startElement as SmkRuleOrCheckpointArgsSection).argumentList ?: return
+            val indent = startElement.prevSibling.text
+            // Moves every argument list element to new line
+            argumentList.arguments.forEach { expression ->
+                doc.insertString(expression.startOffset, "$indent$TAB")
```

Comment:
I think is better to use `indent` instead of TAB here, because actually you need to indent other lines relative to section name statement